hooks/useAttestedVolume.ts

```diff
@@ -0,0 +1,32 @@
+import type { Address } from 'viem';
+
+import { useEthersProvider } from '@/lib/ethers-adapter'
+import { getProxy} from '@/lib/proxy';
+import { useEffect, useState } from 'react';
+import { getAttestedVolume } from '@/lib/volume';
+import { ethers, Wallet } from 'ethers';
+
+
+export function useAttestedVolume(address: Address) {
+  const provider = useEthersProvider()
```

Comment:
Here's what I'm using in aas-frontend to get ethers provider from wagmi: https://github.com/avax-attestations/aas-frontend/blob/main/hooks/useProvider.ts